string ctrl_mode
---
bool success
string message
